packages/app/src/pages/Settings.vue

```diff
@@ -17,4 +17,21 @@ query Settings {
 
 const query = useQuery({ query: SettingsDocument })
 
+gql`
+subscription Settings_cloudProjectUpdate {
+  cloudProjectChange {
+    id
+    ...CloudSettings_CloudProject
+  }
+}
+`
+
+useSubscription({ query: Settings_CloudProjectUpdateDocument }, (current, next) => {
+  // Need to investigate why this is needed. When the subscription comes in the cache should update & automatically
```

Comment:
Is this something you are looking into before this PR is merged? If not, would be great to get an issue logged and linked.

Reply:
Yes, still looking into this. Will log an issue if not

Reply:
I thought I had a similar bug, but the problem was we were caching data in the app layer (in Vue using `ref`). Did you check where this goes and if it's saved/cached anywhere along the line?

Reply:
My fix was around invalidating the spec list caching, landed here: https://github.com/cypress-io/cypress/pull/20744/files#diff-4227699afbcb444e02cc568cf473652e333a9a2847d0164d00e627061772ddd1R191

We need to look into our spec list, it's pretty messy. It was created when a bunch of the team was learning Vue, we know more and can likely do a better job there.